### ISSUE TITLE
Quest Files Can't Export Music or SFX

### ISSUE THREAD

**Issue by ywkls** [ZQuestClassic/ZQuestClassic#40]
One of the most annoying things I've found when creating multiple quests is the fact that I either have to use the same file for all my quests or import each and every music and sfx file that I want to change from the default.

Is there a way to make it where you can export (and import) these files to prevent this hassle?

**Comment by evouga** [comment]
I'll look into it.

**Comment by evouga** [comment]
Sound effects can now be exported. I haven't look into music yet.

I assume you want the ability to export all sounds in a quest to a single aggregated file?

**Comment by ywkls** [comment]
Yeah, like how that currently you can export your entire array of tiles, combos and enemies as one file and import it into another quest. Or back it up, to prevent losing things.

**Comment by ywkls** [comment]
FYI... if I closed this, I didn't know I closed it.

**Comment by ZoriaRPG** [comment]
I will eventually work on the audio libs, and consider how to best do this. I think that one solution is to dump all of the audio files to disc, but a better solution would be a quest template for audio.
